firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -10,7 +10,7 @@ let newTopSite = [
     "topSiteLabel": "Mozilla",
     "bookmarkLabel": "Internet for people, not profit — Mozilla (US)"
 ]
-let allDefaultTopSites = ["Facebook", "YouTube", "Amazon", "Wikipedia", "Twitter"]
+let allDefaultTopSites = ["Facebook", "YouTube", "Amazon", "Wikipedia", "X"]
```

Comment:
Have you found all occurrences of "Twitter"? There may be more.

Reply:
I believe so--I did a quick search and the tests are now passing.

Reply:
There was a use of "Twitter" in `oldHistoryEntries` and some commented out tests. But those did not appear to be causing failures. 🤔